From bd2aac11ba16d433ca45e9bdfba1d67411996d63 Mon Sep 17 00:00:00 2001
From: Linjie Fu <linjie.fu@intel.com>
Date: Thu, 9 Jan 2020 10:17:46 +0800
Subject: [PATCH 34/35] lavc/vaapi_decode: add decode support for
 HEVC_MAIN_STILL_PICTURE

Map FF_PROFILE_HEVC_MAIN_STILL_PICTURE to VAProfileHEVCMain.

Signed-off-by: Linjie Fu <linjie.fu@intel.com>
---
 libavcodec/vaapi_decode.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libavcodec/vaapi_decode.c b/libavcodec/vaapi_decode.c
index 8d5842a..fedb86a 100644
--- a/libavcodec/vaapi_decode.c
+++ b/libavcodec/vaapi_decode.c
@@ -395,6 +395,8 @@ static const struct {
 #if VA_CHECK_VERSION(0, 37, 0)
     MAP(HEVC,        HEVC_MAIN,       HEVCMain    , NULL ),
     MAP(HEVC,        HEVC_MAIN_10,    HEVCMain10  , NULL ),
+    MAP(HEVC,        HEVC_MAIN_STILL_PICTURE,
+                                      HEVCMain    , NULL ),
 #endif
 #if VA_CHECK_VERSION(1, 2, 0)
     MAP(HEVC,        HEVC_REXT,       None,
-- 
2.7.4

